hudi-examples/src/main/java/org/apache/hudi/examples/java/HoodieJavaWriteClientExample.java

```diff
@@ -72,8 +72,11 @@ public static void main(String[] args) throws Exception {
     Path path = new Path(tablePath);
     FileSystem fs = FSUtils.getFs(tablePath, hadoopConf);
     if (!fs.exists(path)) {
-      HoodieTableMetaClient.initTableType(hadoopConf, tablePath, HoodieTableType.valueOf(tableType),
-          tableName, HoodieAvroPayload.class.getName());
+      HoodieTableConfig.propertyBuilder()
+        .setTableType(tableType)
+        .setTableName(tableName)
+        .setPayloadClassName(HoodieAvroPayload.class.getName())
+        .initTable(hadoopConf, tablePath);
```

Comment:
this again makes me feel initTable() should move to HoodieTableMetaClient. bcoz, as you could see here, we start w/ instantiating HoodieTableConfig.propertyBuilder(), but final call results in HoodieTableMetaClient instance which does not sit well.

Reply:
Hi @nsivabalan @yanghua I have moved the `initTable` method and `propertyBuilder` to the `HoodieMetaClient` now. Please take a look again when you have time, Thanks~

Reply:
Hi @nsivabalan ,can we merge this PR? I am preparing another PR to solve the problem in [HUDI-1415](https://github.com/apache/hudi/pull/2283). This PR will add more table properties to the hoodie.properties when initTable. So I hope this PR can merge soon and I can use the `builder` in this PR. Thanks~

Reply:
Agree to move forward and do some subsequence refactor.